com.github.robertsanseries.ciano.json

```diff
@@ -1,16 +1,18 @@
 {
     "app-id": "com.github.robertsanseries.ciano",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "20.08",
+    "runtime-version": "23.08",
     "sdk": "org.freedesktop.Sdk",
-    "command": "com.github.robertsanseries.ciano",
     "base": "io.elementary.BaseApp",
-    "base-version": "juno-20.08",
+    "base-version": "juno-23.08",
+    "command": "com.github.robertsanseries.ciano",
     "finish-args": [
         "--share=ipc",
         "--socket=x11",
         "--socket=wayland",
         "--filesystem=home",
+        "--device=dri",
+        "--env=GTK_THEME=/app/share/themes",
```

Comment:
Setting GTK_THEME does not seem the correct solution. All `GTK_` variables are usually debug variables.

Reply:
Due to the patch implemented, the elementary theme is forced to be used, it seems to me that the theme cannot be located, that's why I added the environment variable. Another solution is to remove the patch.

Reply:
It seems I was wrong and the black window border can be fixed by editing the theme, but I'm not allowed to do so. I think it's because it belongs to io.elementary.BaseApp. Is there a way to do this?